### DISCUSSION TITLE
nwg-shell 0.4.0 & nwg-shell-config 0.4.11 (2023-01-21)

### DISCUSSION THREAD

**Discussion by nwg-piotr** [nwg-piotr/nwg-shell#58] [Changelog]
To simplify packaging for Linux distributions other that Arch*, the Arch-specific 'updates' nwg-panel executor needed to go on retirement. It's just been replaced with the system tray indicator. The `nwg-update-indicator` command is a part of the nwg-shell-config utility.

The script for now supports Arch and derivatives only, but it's open to contributions adding support for other distros. See: https://github.com/nwg-piotr/nwg-shell/discussions/55.

The 'updates' executor will be no longer shipped together with default configs. Current nwg-shell users need to remove it manually from their panels. The `/usr/local/bin/sway-update` script has been depracated, and will be removed sooner or later.

![image](https://user-images.githubusercontent.com/20579136/213840807-64dbae77-437b-4efd-959b-83cfa54f10dd.png)

**Comment by ek9** [comment]
I've only started using `nwg-shell` last week, but this is what bothered me from the start - I had to commend out the `baph` line in both `sway-update` and `sway-check-updates` as I don't use `baph` and it would result in my panel breaking for periods of time with the error `baph command not found`.

How can I disable this all-together (the update check)?

**Reply by nwg-piotr** [reply]
1. Remove the 'updates' executor from you panel(s) (nwg-panel-config):

![image](https://user-images.githubusercontent.com/20579136/213873055-91d4addd-a32a-4352-860f-2ce4f36099f1.png)

2. Disable the tray icon (nwg-shell-config):

![image](https://user-images.githubusercontent.com/20579136/213872999-9a5276ea-06b0-4305-aa2f-82afdb670b70.png)

**Reply by ek9** [reply]
Thank you! I figured there's a better way to disable this - I just postponed digging around in the settings. I plan to customize more later. Some elements in those windows though are very confusing at start UX wise, I hope I'm able to contribute sometime in the future, out of the box config is very usable with minor tweaks

**Reply by nwg-piotr** [reply]
By the way: you may use your own script to perform updates. For checking updates I arbitrary chose baph, as it's very simple and lightweight, and gets me rid of the discussion whether yay is better than trizen and the like. :)
